serde-generate/Cargo.toml

```diff
@@ -16,28 +16,42 @@ exclude = [
 ]
 
 [dependencies]
-heck = "0.3.2"
-include_dir = "0.6.0"
-maplit = "1.0.2"
-serde = { version = "1.0.126", features = ["derive"] }
-serde_bytes = "0.11.5"
-serde_yaml = "0.8.17"
-structopt = "0.3.21"
-textwrap = "0.13.4"
-phf = { version = "0.10", features = ["macros"] }
+serde-generate-cpp = { path = "serde-generate-cpp", optional = true }
+serde-generate-csharp = { path = "serde-generate-csharp", optional = true }
+serde-generate-dart = { path = "serde-generate-dart", optional = true }
+serde-generate-golang = { path = "serde-generate-golang", optional = true }
+serde-generate-java = { path = "serde-generate-java", optional = true }
+serde-generate-ocaml = { path = "serde-generate-ocaml", optional = true }
+serde-generate-python3 = { path = "serde-generate-python3", optional = true }
+serde-generate-rust = { path = "serde-generate-rust", optional = true }
+serde-generate-swift = { path = "serde-generate-swift", optional = true }
+serde-generate-typescript = { path = "serde-generate-typescript", optional = true }
+serde-generate-core = { path = "serde-generate-core" }
 
-serde-reflection = { path = "../serde-reflection", version = "0.3.6" }
-bincode = "1.3.3"
-bcs = "0.1.3"
+[features]
+default = []
```

Comment:
Perhaps all features should be enabled by default to adhere as closely as possible to the existing API, while still allowing consumers to the flexibility to disable default features and only enable their desired languages?